src/core/stream.js

```diff
@@ -735,6 +735,19 @@ var PredictorStream = (function PredictorStreamClosure() {
         buffer[pos] = buffer[pos - colors] + rawBytes[i];
         pos++;
       }
+    } else if (bits === 16) {
+      var bytesPerPixel = colors * 2;
+      for (i = 0; i < bytesPerPixel; ++i) {
+        buffer[pos++] = rawBytes[i];
+      }
+      for (; i < rowBytes; i += 2) {
+        var sum = ((rawBytes[i] & 0xFF) << 8) +
+                  (rawBytes[i + 1] & 0xFF) +
+                  (((buffer[pos - bytesPerPixel] & 0xFF) << 8) +
+                  (buffer[pos - bytesPerPixel + 1] & 0xFF));
```

Comment:
Could you add one more space the indenting of this line? Because of the many parentheses, it's easier to see that way that this line is a part of the expression on the previous line.

Reply:
Which line? The first? Or all except the first? Or add spaces so that "rawbytes" and "buffer" are aligned? The green block here has 4 lines.

Reply:
Line 747. GitHub adds the comment at the correct line, but shows other lines for context. You would end up with:

```javascript
        var sum = ((rawBytes[i] & 0xFF) << 8) +
                  (rawBytes[i + 1] & 0xFF) +
                  (((buffer[pos - bytesPerPixel] & 0xFF) << 8) +
                   (buffer[pos - bytesPerPixel + 1] & 0xFF));
```

Reply:
Now that I look at it again, do we actually need those last two parentheses? I think this would be just fine:

```javascript
        var sum = ((rawBytes[i] & 0xFF) << 8) +
                  (rawBytes[i + 1] & 0xFF) +
                  ((buffer[pos - bytesPerPixel] & 0xFF) << 8) +
                  (buffer[pos - bytesPerPixel + 1] & 0xFF);
```

Reply:
The copyright is here at the last page:
http://www.gwg.org/wp-content/uploads/GWG_PDF-Output-Suite4_ReadMeFirst2.pdf

Copyright © 2014, Ghent Workgroup (http://www.gwg.org). All Rights Reserved. The Ghent 
Workgroup hereby grants permission to use this test suite and its documentation as described 
in the associated documentation, subject to the following conditions. This legal notice must be 
included in all copies containing the whole or substantial portions of the Ghent PDF Output 
Suite. Without express written permission of the Ghent Workgroup it is not permitted to use this 
PDF Output Suite for anything but its intended purpose of testing workflow setup. The Ghent 
PDF Output Suite cannot be sold or used in any commercial context without previous written 
permission by the Ghent Workgroup.

So it's a bit tricky, the copyright notice would have to be added too somehow, so I guess it's better to use the link.

Reply:
Yes, let's indeed use a linked test case.